ULC/Repl.hs

```diff
@@ -51,7 +51,7 @@ settings = Settings contextCompletion Nothing True
     completions :: String -> String -> StateT Environment IO [Completion]
     completions prev xs = if "daol:" `isPrefixOf` dropWhile isSpace prev -- check if previous word (in reverse) is ":load"
                             then listFiles xs
-                            else map (\x -> Completion x x True) . filter (xs `isPrefixOf`) . (comms ++) . M.keys <$> get
+                            else map (\x -> Completion x x True) . filter (xs `isPrefixOf`) . (if all isSpace prev then (comms ++) else id) . M.keys <$> get
```

Comment:
This is quite the long line, please can you split this up into something shorter